client/src/components/AppHeader/AppHeader.js

```diff
@@ -45,13 +44,11 @@ class AppHeader extends React.Component {
     }
 
     render() {
-        console.log('appheader props', this.props)
         const { user, isAuthenticated } = this.props;
         const { mobileMenuOpened, activePage } = this.state;
         const avatarURL = user && user.avatar ? 
             <img src={user.avatar} alt={user.email}/> 
-            : <img src={face} alt ="user" />
-        console.log('[appheader user]', user)
+            : <img src={face} alt ="user" />;
```

Comment:
Better add user name or just leave empty alt attribute. Not critical, but if possible, enhance that moment pls.

Reply:
Ok! I'll do it in the next pull request